assets/app/scripts/app.js

```diff
@@ -121,21 +121,42 @@ angular
       .when('/project/:project/browse/builds', {
         templateUrl: 'views/builds.html'
       })
+      .when('/project/:project/browse/builds/:buildconfig', {
```

Comment:
I can never remember... do we want a singular path segment for a specific item (e.g. `browse/build/<mybuild>`) in order to save subpaths of `builds` for actions (e.g. `builds/new`)?